contributor.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# This script is an alternative method for not having to deal
+# with GitHub Package Registry.
+# 
+# By calling this script, the latest source code of gctoolkit-testdata 
+# will be downloaded, and the project will be built and installed in the
+# local Maven repository.
+# 
+# This script will also enable the 'contributor' profile to ensure all 
+# tests and checks pass, since the GC logs will be available.
+
+# Turn on the Maven Config file with Contributor Profile activated
+activate() {
+    if [ -f .mvn/maven.config.contributor ]; then
+        cp .mvn/maven.config.contributor .mvn/maven.config
+    fi
+    echo "Maven 'contributor' profile activated. See ./mvn/maven.config"
+}
+
+revert() {
+    if [ -f .mvn/maven.config ]; then
+        rm .mvn/maven.config
+    fi
+    echo "./mvn/maven.config file removed."
+}
+
+# Download latest GCToolkit Test Data pack
+download() {
+    rm -rf gctoolkit-testdata
```

Comment:
You always want to clean?

Reply:
Pretty much, given this is a temporary folder. I will actually rename it with a period to hide in *nix systems.